addons/website/views/snippets/snippets.xml

```diff
@@ -1262,19 +1255,23 @@
                         data-null-value="'NULL'"
                         data-with-combinations="customizeWebsiteColor"
                         data-with-gradients="true"/>
-        <we-select data-name="footer_language_selector_opt" string="Language Selector" data-reload="/">
+        <div> 
+            <!-- div wrapper to avoid a CSS declaration forcing the margin at 12px added by this commit : 
+                https://github.com/odoo/odoo/commit/838564e0acf2f1c2e1727473c8804c010aebae2d -->
```

Comment:
I don't see the issue ... I compared the options with and without your change and I cannot see the difference ...
Maybe the commit message is not clear enough to understand what is fixed here ;)

Reply:
@bvr-odoo I understand why. It was fixed after this PR with this commit : https://github.com/odoo/odoo/commit/9bd063cd2315da5f4de0921e4aca921b150b5e82